.gitlab-ci-docker-build-multiplatform.yml

```diff
@@ -0,0 +1,93 @@
+variables:
```

Comment:
We are missing documentation like the rest of the templates.

Reply:
Thought there was no need since the variable section is pretty self documenting.

Reply:
Sure. But then we should update the rest of the templates or, at a minimum, create a follow-up task for it.